arctic/store/_pickle_store.py

```diff
@@ -54,12 +56,16 @@ def write(self, arctic_lib, version, symbol, item, previous_version):
         collection = arctic_lib.get_top_level_collection()
         # Try to pickle it. This is best effort
         version['blob'] = _MAGIC_CHUNKED
-        pickled = lz4.compressHC(cPickle.dumps(item, protocol=cPickle.HIGHEST_PROTOCOL))
+        try:
+            pickled = lz4.compressHC(cPickle.dumps(item, protocol=cPickle.HIGHEST_PROTOCOL))
+        except OverflowError:
```

Comment:
that shouldnt be a problem. LZ4 bypass will only happen if you get an overflow error. If that DOES happen, you set the type to _MAGIC_CHUNKED_NO_COMPRESS, so when you read it back, it will not decompress. If you move to Lz4x64, you wont have an overflow error, and thus will bypass all this new code.